.travis.yml

```diff
@@ -14,5 +14,5 @@ before_script:
 script:
   - cargo fmt --all -- --check
   - cargo test --no-default-features
-  - cargo test --all-targets --no-default-features --features alloc
+  - cargo test --no-default-features --features alloc
```

Comment:
What Why did the `--all-targets` flag had to be removed here?

Reply:
Because benches use `std` and can't be built with just `alloc`